### PR TITLE
Rebase 1.21.0 rc.0

### PR DESCRIPTION
**Pull request by soltysh**
Tracking sheet https://docs.google.com/spreadsheets/d/10KYptJkDB1z8_RYCQVBYDjdTlRfyoXILMa0Fg8tnNlY/edit#gid=1904553197

### CODE REVIEW

**Comment by soltysh** [comment]
/test e2e-aws-fips
fails on pulling `registry.redhat.io/redhat/redhat-marketplace-index:v4.7` and `registry.redhat.io/redhat/certified-operator-index:v4.7`

**Comment by soltysh** [comment]
`Image signature workflow can push a signed image to openshift registry and verify it` in `ci/prow/e2e-aws-serial` is disabled in https://github.com/openshift/origin/pull/26047

**Comment by soltysh** [comment]
/override ci/prow/verify-commits 
this won't pass on a k8s bump pr

**Comment by sttts** [comment]
Tests were green, but we needed a rebase onto master due to merge conflicts.

/lgtm
/approve

File: openshift-hack/e2e/annotate/generated/zz_generated.annotations.go [hunk 1]
```diff
@@ -707,7 +707,7 @@ var annotations = map[string]string{
 
 	"[Top Level] [sig-network] Conntrack should be able to preserve UDP traffic when server pod cycles for a NodePort service": "should be able to preserve UDP traffic when server pod cycles for a NodePort service [Suite:openshift/conformance/parallel] [Suite:k8s]",
 
-	"[Top Level] [sig-network] Conntrack should drop INVALID conntrack entries": "should drop INVALID conntrack entries [Suite:openshift/conformance/parallel] [Suite:k8s]",
+	"[Top Level] [sig-network] Conntrack should drop INVALID conntrack entries": "should drop INVALID conntrack entries [Disabled:Broken] [Suite:k8s]",
```
Comment (smarterclayton) [inline openshift-hack/e2e/annotate/generated/zz_generated.annotations.go:710]: looks scary, is it?

Comment (soltysh) [reply openshift-hack/e2e/annotate/generated/zz_generated.annotations.go:710]: Opened https://bugzilla.redhat.com/show_bug.cgi?id=1945329 for networking team to investigate.

**Pull request merged**